### ISSUE TITLE
Logout when PL ends

### ISSUE THREAD

**Issue by MDA999** [mixxxdj/mixxx#12643]
### Feature Description

I would like to see an option, to automatically logout of the livestream, when the playlist ends, so someone else can take over the stream at a specific time, while the active streamer is away.

Also, the feature to get the time of end of the playlist is a wished and confirmed feature for nearly 2 years, still seems not implemented. Would be good to have these essential features for a stream, that is managed by more than 1 person.

**Comment by ronso0** [comment]
I presume you're referring to the AutoDJ playlist.
Sounds like a valid feature request.

Sure broadcasting can connect to `[AutoDJ],enabled` in some way. Such an option can be added to the Preferences, but making this option accessible in the main GUI would be much better.
This would yet another argument to turn the Broadcasting/"On Air" toggle into a **menu button¹** which contains
* toggle and status display for each broadcast connection
* options when to start/stop broadcasting (at fixed time, when AutoDJ starts/stops (last loaded track finishes), after N seconds of silence and what not)

¹much like the effect chain preset button in Mixxx 2.4 
![image](https://github.com/mixxxdj/mixxx/assets/5934199/f71f5d8c-29ae-4c82-abd1-1cdf6a1d6962)

Related:
#10311 
#10434 
#9931 
#9631 
#10188

**Comment by MDA999** [comment]
Yes, and I guess it could also come handy sometimes to add a timed logoff of the livestream. Like: set the timer for Livestream Logout for a specific system time, so the other streamer can be sure he can overtake the stream at that time.
